include/operations/blas3_trees.hpp

```diff
@@ -26,12 +26,12 @@
 #ifndef BLAS3_TREES_HPP
 #define BLAS3_TREES_HPP
 
-#include <iostream>
 #include <stdexcept>
 #include <vector>
+#include <iostream>
```

Comment:
We shouldn't include iostream on the tree files, is it that we have some debug cout?